crest/Assets/Crest/Crest/Scripts/Shapes/OceanWaveSpectrum.cs

```diff
@@ -87,11 +87,11 @@ public float GetAmplitude(float wavelength, float componentsPerOctave)
             return a;
         }
 
-        float ComputeWaveSpeed(float wavelength)
+        public static float ComputeWaveSpeed(float wavelength, float gravityMultiplier = 1f)
```

Comment:
To make this consistent with the implementation in the shader, it might be better to have the second parameter just be the value of gravity in ms^2 instead of a multiplier of Earth's gravity.

Reply:
discussed on discord